test/controllers/resource_controller_test.rb

```diff
@@ -191,4 +191,40 @@ class ResourceControllerTest < ActionDispatch::IntegrationTest
       post api_namespace_resource_index_url(api_namespace_id: api_namespace.id), params: payload
     end
   end
+
+  test 'should allow #create with the current_user & current_visit being available in the ' do
```

Comment:
'should allow #create with the current_user & current_visit being available in the send_web_request action'